### PR TITLE
Fix Mobile Tests

### PR DESCRIPTION
**Pull request by budzworthy**
## Description

This PR fixes and improves all of the e2e integration tests for the mobile configuration

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] User stories met
- [ ] Storybook stories
- [ ] Unit tests added
- [x] Linting passing
- [x] Unit tests passing
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [x] UX tested
- [x] Browsers / platforms tested
- [x] Rebased & ready to merge without conflicts
- [x] Reviewed own code

### CODE REVIEW

File: test-e2e/tests/video_detail_mobile_test.ts [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 
+const agent327Title = 'Agent 327';
+const agent327Description = 'Hendrik IJzerbroot – Agent 327 – is a secret agent working for the Netherlands secret service agency. In the twenty comic books that were published since 1968, Martin Lodewijk created a rich universe with international conspiracies, hilarious characters and a healthy dose of Dutch humour.';
+
 Scenario('Video detail screen loads', ({ I }) => {
   I.amOnPage('http://localhost:8080');
-  I.scrollTo({ css: 'div[aria-label="Play Agent 327"]'});
-  I.click({ css: 'div[aria-label="Play Agent 327"]'});
+  openVideo(I, 'Agent 327');
   I.see('Agent 327');
```
Comment (rcawkwell) [inline test-e2e/tests/video_detail_mobile_test.ts:12]: Would be nice use the `agent327Title`

File: test-e2e/utils/steps_file.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  let jwplayer: () => {getState: () => string};
+}
+
+module.exports = function() {
+  return actor({
+
+    // Define custom steps here, use 'this' to access default methods of I.
+    // It is recommended to place a general 'login' function here.
+
+    login: function (this: CodeceptJS.I, email = '12345@test.org', password = 'Ax854bZ!$') {
```
Comment (rcawkwell) [inline test-e2e/utils/steps_file.ts]: Can we use the constants.username and password here?

**Review by rcawkwell — APPROVED** [review]
Looks good. Left a couple small comments.

File: test-e2e/utils/steps_file.ts (hunk 2 above)
Comment (budzworthy) [reply test-e2e/utils/steps_file.ts]: Done

File: test-e2e/tests/video_detail_mobile_test.ts (hunk 1 above)
Comment (budzworthy) [reply test-e2e/tests/video_detail_mobile_test.ts:12]: Thanks, I think that was the intention but then I never circled back to it. Fixed in latest.

**Pull request merged**